sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -14,13 +15,34 @@ namespace Azure.Monitor.Ingestion
 {
     /// <summary> The IngestionUsingDataCollectionRules service client. </summary>
     [CodeGenClient("IngestionUsingDataCollectionRulesClient")]
+    [CodeGenSuppress("LogsIngestionClient", typeof(Uri), typeof(TokenCredential), typeof(LogsIngestionClientOptions))]
```

Comment:
I am assuming this was added manually. Is there a way to not add these attributes in the public layer but perhaps put them in the autorest file instead? This feels like a knob we are using that is not supported by default on DPG
/cc: @m-nash